### DISCUSSION TITLE
Possible to change UI color without changing document colors?

### DISCUSSION THREAD

**Discussion by jwong00** [sumatrapdfreader/sumatrapdf#2409] [Q&A]
When I set `UseSysColors = true`, the bookmarks menu will use my system colors, but so will the PDF. In previous versions, I did not need to set this to `true` just to get the bookmarks sidebar to respect system colors. If there is currently a way to do set my UI settings to fully respect system colors without modifying the document colors, how do I do so?

With `UseSysColors = true`: 

![mQ6Q1nv07z](https://user-images.githubusercontent.com/8384248/157327245-5da2dc27-656a-4d0a-8160-bc1abc0d23fe.png)

With `UseSysColors = false`: 
![bI9mHQ311e](https://user-images.githubusercontent.com/8384248/157327269-3f803a7a-fe30-4c91-b16b-cc2cb30db3a6.png)

**Comment by kjk** [comment]
The big picture answer is: theming (changing colors) has gaps, as you see.

If you can't get what you want using documented settings https://www.sumatrapdfreader.org/settings/settings then it cannot be done currently.

`UseSysColors` was a mistake. Infinite possibilities make it impossible for me to test that a feature works 100%.

Putting more work into theming is low on my priority list.

**Reply by jwong00** [reply]
Fair enough. For anyone else reading who's having this issue, I've found that reverting to a slightly older version (3.1.2) does "solve" the issue. Thanks for getting back.
![SumatraPDF_ZIB5X2bc05](https://user-images.githubusercontent.com/8384248/157331792-35f9c01b-c566-4e7c-95ca-c8a212ece065.png)
